docker-compose.yaml

```
@@ -0,0 +1,9 @@
+version: '3.1'
+
+services:
+  banking-api-db:
+    image: postgres:13.2-alpine
+    environment:
+      POSTGRES_PASSWORD: postgres
```

É uma boa prática trocar o usuário e senha default do postgresql, isso evita mascarar problemas no setup da aplicação.

```suggestion
      POSTGRES_USER: banking_api_user
      POSTGRES_PASSWORD: banking_api_pass
```